src/imp/libc/time/syscalls.rs

```diff
@@ -73,3 +80,35 @@ pub(crate) fn clock_gettime_dynamic(id: DynamicClockId<'_>) -> io::Result<Timesp
         Ok(timespec.assume_init())
     }
 }
+
+#[cfg(any(target_os = "android", target_os = "fuchsia", target_os = "linux"))]
```

Comment:
Huh, interesting that Fuschsia cloned this one.  I guess like FreeBSD in some cases, they felt it was easier for ecosystem adoption to just use the Linux API.